src/modules/addremove.js

```diff
@@ -1,14 +1,6 @@
 /* eslint-disable import/no-cycle */
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, It would be better to **remove the comments to disable linter checks.** The linter checks are here to help you improve the quality of your code 🙏 Kindly remove all of them as suggested